vulkaninfo/generated/vulkaninfo.hpp

```diff
@@ -5212,4 +5212,18 @@ bool operator==(const VkSurfaceFormatKHR & a, const VkSurfaceFormatKHR b) {
 std::ostream &operator<<(std::ostream &o, VkExtent3D &obj) {
     return o << "(" << obj.width << ',' << obj.height << ',' << obj.depth << ")";
 }
+auto format_ranges = std::array{
```

Comment:
FFTI: Add a comment explaining the contents of this array. The contents of this array aren't immediately apparent.

Reply:
I would add a comment but doing so would make the rest of the code based odd choices undercommented in comparison. Rather I think the concept of a format range isn’t too complex as well as the struct definitions own comment’s explaining its existence.